install/source/chapter11.rst

```diff
@@ -10,8 +10,8 @@ __ CentreStack_Blog_
 Using the GUI Installer to Upgrade
```

Comment:
Check the first paragraph, seems the link in the CentreStack Blog got lost at some point, can you fix it?

Reply:
Ok, so I have identified where the problem is. It seems like the issue was there even in the master build, so I'm just going to try to fix it.